test/utf8_test.dart

```diff
@@ -2,13 +2,76 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:ffi';
+import 'dart:typed_data';
+
 import 'package:test/test.dart';
 import 'package:ffi/ffi.dart';
 
+Pointer<Uint8> _bytesFromList(List<int> ints) {
+  Pointer<Uint8> ptr = Pointer.allocate(count: ints.length);
```

Comment:
Prefer final variables if they are not re-assigned

Reply:
For the record: That's neither recommended nor discouraged by the style guide, just something some projects may choose to do.

Reply:
Done.